applications/application_template.md

```diff
@@ -1,128 +0,0 @@
-# Crust Grant Proposal
-
-* **Project Name:** Name of your project 
-* **Team Name:** Legal name of your team
-* **Payment Address:** Ethereum address 
-* **Receivable Token:** CRU or USDT
-
-> ⚠️ *The combination of your GitHub account submitting the application and the payment address above will be your unique identifier during the program. Please keep them safe.*
-
-> ⚠️ *When you submit an application for a Grant，You confirm that you have read these Terms and Conditions carefully and that you agree to be bound by them.*
-
-### Overview
```

Comment:
You can't remove or change the original file, this is described in [Application Process](https://github.com/crustio/Crust-Grants-Program#1-application) clearly right?